project.php

```diff
@@ -1223,8 +1223,8 @@ function do_holds()
     echo "<table>\n";
     echo "<tr>\n";
     echo "<th></th>\n";
-    echo "<th style='padding: 0em 1em'>", _("hold in Waiting"), "</th>\n";
-    echo "<th style='padding: 0em 1em'>", _("hold in Available"), "</th>\n";
+    echo "<th>", _("hold in Waiting"), "</th>\n";
+    echo "<th>", _("hold in Available"), "</th>\n";
```

Comment:
After looking at this and comparing it to what Event Subscriptions already was, I think it will work fine without the padding. I would definitely expect some people to notice it, though, since the two table headers are crammed right up against each other, without even a punctuation mark.

Reply:
<nod> I added a second commit to put some spacing between the columns on both of these tables.